.github/workflows/unit-test.yml

```
@@ -11,17 +11,17 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     timeout-minutes: 15
-
     steps:
     - uses: actions/checkout@v2
```

actionlint (1.7.9)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)